modules/reference/pages/systems/system-details/sd-ansible.adoc

```diff
@@ -0,0 +1,19 @@
+[[ref-systems-sd-ansible]]
+= Ansible
+
+The menu:Systems[Ansible] section allows you to use the Ansible features on the Control Node system.
+
+The tab is split into the following subtabs:
+
+Control Node::
+Place for defining the paths to inventory files and playbook directories on your Ansible Control Node.
+
+Playbooks::
+Lists all playbooks in the defined paths. You can click a playbook to display its details or to schedule its execution.
+
+Inventories::
+Lists all inventory files in the defined paths. Clicking an inventory file in the list introspects its contents.
+
+
+For more information, refer to xref:administration:ansible-integration.adoc[].
```

Comment:
```suggestion
For more information, see to xref:administration:ansible-integration.adoc[].
```
I'm going to accept this suggestion myself.

Reply:
:+1: